travis.sh

```diff
@@ -147,6 +147,11 @@ if [ "$BUILDER" == catkin ]; then catkin build -i -v --limit-status-rate 0.001 $
 travis_time_end
 travis_time_start catkin_run_tests
 
+# patch for rostest
+(cd /opt/ros/$ROS_DISTRO/lib/python2.7/dist-packages; wget --no-check-certificate https://patch-diff.githubusercontent.com/raw/ros/ros/pull/82.diff -O - | sudo patch -p4 )
```

Comment:
patching is only required for hydro
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/trusty-travis-jsk-ros-pkg-jsk_travis-indigo-deb-true/591/console